src/react-dnd-ax/drag-n-drop-item.js

```diff
@@ -53,6 +53,7 @@ const DragNDropItem = (WrappedComponent) => {
         })
         this.dragPointElem.addEventListener('click', this.onClick)
         this.dragPointElem.addEventListener('keyup', this.onEnter)
+        this.dragPointElem.addEventListener('keyup', this.onSpace)
```

Comment:
how about combine the logic into onEnter callback, so we don't have to add a new listener onSpace. Just a thought , either way is fine by me.

Reply:
Thanks Viking. It is small enough that I think I will leave it as-is for now, but if we need to add another listener or it becomes a problem, then perhaps we can think about combining!